const errorHandleMiddleware = store => next => action => {
  try {
    console.log("ðŸ˜€ Success");
    next(action);
  } catch (err) {
    console.error("ðŸ˜ŸError is ", err);
  }
};

exports.errorHandleMiddleWare = errorHandleMiddleware;
